conda/recipes/custreamz/recipe.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2018-2025, NVIDIA CORPORATION.
+schema_version: 1
+
+context:
+  version: ${{ env.get("RAPIDS_PACKAGE_VERSION") }}
+  minor_version: ${{ (version | split("."))[:2] | join(".") }}
+  cuda_version: ${{ (env.get("RAPIDS_CUDA_VERSION") | split("."))[:2] | join(".") }}
+  cuda_major: '${{ (env.get("RAPIDS_CUDA_VERSION") | split("."))[0] }}'
+  date_string: '${{ env.get("RAPIDS_DATE_STRING") }}'
+  py_version: ${{ env.get("RAPIDS_PY_VERSION") }}
+  py_buildstring: ${{ py_version | version_to_buildstring }}
+  head_rev: ${{ git.head_rev(".")[:8] }}
+
+package:
+  name: custreamz
+  version: ${{ version }}
+
+source:
+  path: ../../..
+
+build:
+  string: cuda${{ cuda_major }}_py${{ py_buildstring }}_${{ date_string }}_${{ head_rev }}
+  script:
+    content: |
+      ./build.sh custreamz
+
+requirements:
+  host:
+    - python =${{ py_version }}
+    - pip
+    - rapids-build-backend >=0.3.0,<0.4.0.dev0
+    - setuptools
+    - python-confluent-kafka >=2.5.0,<2.6.0a0
+    - cudf_kafka =${{ version }}
+    - cuda-version =${{ cuda_version }}
+  run:
+    - python
+    - streamz
+    - cudf =${{ version }}
+    - cudf_kafka =${{ version }}
+    - rapids-dask-dependency =${{ minor_version }}
+    - python-confluent-kafka >=2.5.0,<2.6.0a0
```

Comment:
I don't know anything about the ABI stability of this library but we might want to at least consider opening an issue asking if it should have run exports for us.